secretsmanager/src/secrets_manager_enc.cpp

```diff
@@ -76,28 +83,31 @@ std::optional<std::string> SecretsManagerEnc::decryptFile(const std::ifstream& f
 std::optional<std::string> SecretsManagerEnc::decryptString(const std::string& input) { return decrypt(input); }
 
 std::optional<std::string> SecretsManagerEnc::decrypt(const std::string& data) {
-  try {
-    // AES_CBC is created on each call fir thread safety
-    auto aes = AES_CBC(*key_params_);
-    auto cipher_text = base64Dec(data);
-    auto pt = aes.decrypt(cipher_text);
-
-    return std::optional<std::string>{pt};
-  } catch (std::exception& e) {
-    LOG_ERROR(logger_, "Decryption error: " << e.what());
+  if (!data.empty()) {
```

Comment:
for less identation you can do:
if (data.empty()) {
return;
}
try {
....
}

Reply:
Done.